docs/cryptodoc/src/conf.py

```diff
@@ -12,10 +12,9 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-import os
-import sys
-sys.path.insert(0, os.path.abspath('../../_ext'))
-
```

Comment:
No more hard-coded import into a relative path to gain access to the `:srcref:` extension. The relative path dependency is now (much more prominently) defined in the `pyproject.toml`.

This will facilitate code reuse across document generators. E.g. to have a central configuration (e.g. for the Botan version, Authors, GitHub references and so on).